spring-integration-core/src/main/resources/META-INF/spring/aot.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.aot.hint.RuntimeHintsRegistrar=org.springframework.integration.aot.CoreRuntimeHintsRegistrar
```

Comment:
The cookbook says `spring.aot.factories` - which is correct?

Reply:
AFAIK under `aot.factories` file `spring/` dir. See Spring Web for example: https://github.com/spring-projects/spring-framework/blob/main/spring-web/src/main/resources/META-INF/spring/aot.factories

Reply:
Yes, it's `META-INF/spring/aot.factories`. The one in the cookbook is a typo.